onebusaway-android/build.gradle

```diff
@@ -27,11 +27,14 @@ apply plugin: 'com.android.application'
 
 repositories {
     jcenter()
-    mavenLocal()
     maven {
         // OBA Releases - for comparator to sort alphanumeric routes
         url "http://nexus.onebusaway.org/nexus/content/repositories/releases"
```

Comment:
For the time being I think this is good though. Since this just moves the existing files it isn't actually bloating the repo at all.